pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/models/TopKBundles.java

```diff
@@ -66,7 +66,7 @@ public void update(Map<String, NamespaceBundleStats> bundleStats, int topk) {
         topk = Math.min(topk, arr.size());
         partitionSort(arr, topk);
 
-        for (int i = 0; i < topk; i++) {
+        for (int i = topk - 1; i >= 0; i--) {
```

Comment:
Why reverse the topK bundles order?

Reply:
We want to transfer lower-loaded ones first to meet the target threshold. If the highest is too much traffic, we can't unload at all.